perf_test/blas/blas1/KokkosBlas_team_dot_perf_test_benchmark.cpp

```diff
@@ -128,8 +128,6 @@ static void run(benchmark::State& state) {
     double avg   = total / repeat;
     // Flops calculation for a 1D matrix dot product per test run;
     size_t flopsPerRun = (size_t)2 * m;
-    printf("Avg DOT time: %f s.\n", avg);
-    printf("Avg DOT FLOP/s: %.3e\n", flopsPerRun / avg);
     state.SetIterationTime(timer.seconds());
```

Comment:
I would move this change to a separate PR ideally

Reply:
Done (all blas1 modifications reverted).